packages/tasks/src/model-libraries.ts

```diff
@@ -95,6 +95,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.audiocraft,
 		filter: false,
 	},
+	audioseal: {
+		prettyLabel: "Audioseal",
+		repoName: "audioseal",
+		repoUrl: "https://github.com/facebookresearch/audioseal",
+		filter: false,
+		countDownloads: { wildcard: { path: "*.pth" } },
```

Comment:
won't this double count downloads?

Reply:
Not quite, only one model is used at a time, either the generator or the detector. They both also have different code snippets, maybe I should add both to the snippets. Let moi add.

Reply:
Fixed in the latest commit. I think it should be fine since both checkpoints are not used in one go.